pkg/loader/util.go

```diff
@@ -448,3 +449,18 @@ func getOracleUniqKeys(db *gosql.DB, schema, table string) (uniqueKeys []indexIn
 
 	return
 }
+
+func removeDDLPlacementOptions(sql string) (string, error) {
+	stmt, err := parser.New().ParseOneStmt(sql, "", "")
+	if err != nil {
+		return "", err
+	}
+
+	var sb strings.Builder
+	restoreCtx := format.NewRestoreCtx(format.DefaultRestoreFlags|format.RestoreTiDBSpecialComment|format.SkipPlacementRuleForRestore, &sb)
```

Comment:
Can we check whether to change DDL by parser checking does this DDL contains `placementRule`?
Maybe most DDLs don't contain placement rules. But after this commit, their DDLs will be all rewritten.

Reply:
The simplest way is to check if string contains the string `/*T![placement]`